README.md

```diff
@@ -55,6 +56,26 @@ Groups can be added manually or using the group.py script
 
 ![](CCV-auto-group.png)
 
+6. Acknowledging vulnerabilities in bulk
+
+6.1 --export-affected-devices: use ack_vuln.py with export-affected-devices options to export vulnerable devices into CVS file. This script takes two additional parameters \
+a. --cve-id [mandatory] \
+b. --export-affected-devices-preset-name  [optional] if this value is not provided, it takes "All data" as default value \
+E.g., ``` python3 ack_vuln.py --export-affected-devices --cve-id=CVE-2014-0317 ```
+
+6.2 --ack-by-preset: use ack_vuln.py with ack-by-preset options to acknowledge vulnerable devices by preset name and cve. This script takes three additional parameters \
+a. --cve-id [mandatory] \
+b. --ack-by-preset-preset-name  [optional] if this value is not provided, it takes "All data" as default value \
+c. --ack-comments [optional] if this value is not provided, it takes default value of "Default acknowledge comment"
+E.g., ``` python3 ack_vuln.py --ack-by-preset --cve-id=CVE-2014-0317 --ack-comments="Testing Bulk Ack from Python"```
+
+6.3 --ack-by-devices: use ack_vuln.py with ack-by-devices options to acknowledge vulnerable devices that were exported in first options. This script takes additional parameters \
+a. --cve-id [mandatory] \
+b. --ack-comments [optional] if this value is not provided, it takes default value of "Default acknowledge comment"
+ 
+E.g., ``` python3 ack_vuln.py --ack-by-devices --cve-id=CVE-2014-0317 --ack-comments="Testing Bulk Ack from Python"```
+
+Note: Above script does not refresh preset, if you need latest data then kindly refresh preset from GUI and then run above scripts. 
```

Comment:
if you need latest data, then, kindly refresh preset from GUI or API and then run the above script.